aerospike/metadata.csv

```diff
@@ -371,7 +390,7 @@ aerospike.namespace.batch_sub_read_success,gauge,,,,The number of successful bat
 aerospike.namespace.batch_sub_read_timeout,gauge,,,,The number of batch-index read sub transactions that timed out.,0,aerospike,,
 aerospike.namespace.batch_sub_tsvc_error,gauge,,,,The number of batch-index read sub transactions that failed with an error in the transaction service before attempting to handle the transaction.,0,aerospike,,
 aerospike.namespace.batch_sub_tsvc_timeout,gauge,,,,The number of batch-index read sub transactions that timed out in the transaction service before attempting to handle the transaction.,0,aerospike,,
-aerospike.namespace.client_delete_error,gauge,,,,The number of client delete transactions that failed with an error.,0,aerospike,,
```

Comment:
Rather than changing this metric, make a new metrics called `aerospike.namespace.client_delete_error.count` and label that `[OpenMetricsV2]` while labeling `aerospike.namespace.client_delete_error` with `[Legacy]`.

Reply:
got it thank you! I will do the same for a couple others I removed.